### PR TITLE
Fixes #122: Adds release_maintainer.sh script

### LINKED ISSUE #122: Better Deployment for Meilix Generator
Currently, we are deploying on Github only due to which we have a collection of many builds and the pile keep on increasing and requires a cleaning from time to time so to avoid that we can shift from GitHub deployment to [transfer.sh](https://transfer.sh/) where we can host the build for 14days and that can be used for the time period of the event also for which the build is required

### PR DESCRIPTION
**Pull request by abishekvashok**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [ ] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message` >> The issue number is in the PR and the commits should be squashed and merged cause I am on the web.

Adds release_maintainer.sh script which will delete old releases (releases which are more than 10 days old)

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

Fixes #122

### CODE REVIEW

**Comment by abishekvashok** [comment]
@fossasia/linux-devops-admin please review

File: scripts/release_maintainer.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -e
+
+# jq is the JSON parser we will be using
+sudo apt-get install jq
```
Comment (tabesin) [inline scripts/release_maintainer.sh]: -y

File: scripts/release_maintainer.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+current_day=`date +%d`   # Current date eg) 24
+
+for i in $response; do
+    if [ $((index % 2)) -eq 0 ]; then # We get the published_date of the release as $i's value here
+        published_year=${i:1:4}
+        published_month=${i:6:2}
+        published_day=${i:9:2}
+
+        if [ $published_year -lt $current_year ]; then
+             let "delete=1"
```
Comment (tabesin) [inline scripts/release_maintainer.sh]: Not the best style but...

**Comment by tabesin** [comment]
Better trigger the cleanup-script after each new build.

**Comment by xeon-zolt** [comment]
will it also delete the published release by master ?

**Comment by abishekvashok** [comment]
> will it also delete the published release by master ?

Yes

**Comment by meets2tarun** [comment]
If we are able to take a decision on the number of days to decide the ISO that much old, that wouldn't be a bad idea.

**Comment by abishekvashok** [comment]
> If we are able to take a decision on the number of days to decide the ISO that much old, that wouldn't be a bad idea.

I have set days

@fossasia/linux-devops-admin pls add an encrypted variable in the travis dashboard: An access token with delete_pushes scope

Also, if you could modify the pr titles to contain a certain keyword, we could keep master builds.

**Comment by meets2tarun** [comment]
@abishekvashok 
There is only delete_repo scope.

**Comment by abishekvashok** [comment]
I meant full repository permissions.

**Comment by meets2tarun** [comment]
that is already added with the term `KEY`

**Comment by tabesin** [comment]
The codacy checks look garbage. Anyway, personally I think the script should get documented properly and named properly but there is no risk to put it in the repo now as it is w/o access key anyway.

**Comment by abishekvashok** [comment]
@tabesin what more do you expect --- the doc.
And the key is not provided there as mentioned in the pr body.

**Pull request merged**